lib/engine/game/base.rb

```diff
@@ -1935,7 +1939,7 @@ def game_corporations
 
       def init_hexes(companies, corporations)
         blockers = {}
-        companies.each do |company|
+        blocker_companies.each do |company|
```

Comment:
why don't you just change this to be companies + corporations

Reply:
See above.

Reply:
Removing.